client/src/store.ts

```diff
@@ -123,7 +122,7 @@ const persistConfig = {
 
 export const configureStore = () => {
   const store = createStore(
-    persistReducer(
+    persistReducer<any>(
```

Comment:
This was giving me compile errors and I couldn't figure out a proper type after quite a bit of effort, so just went with `any` for now.